pkg/server/proto/server.proto

```diff
@@ -4705,6 +4705,9 @@ message Pipeline {
       // Built-in up operation
       Up up = 8;
 
+      // A reference to a different pipeline
```

Comment:
```suggestion
      // A reference to an embedded pipeline
```
But also, why do we include the entire pipeline object here instead of just a ref? It would make GetPipeline endpoints return unnecessarily heavy content, no?